wrench/src/reftest.rs

```diff
@@ -19,6 +19,15 @@ use webrender::api::*;
 use wrench::{Wrench, WrenchThing};
 use yaml_frame_reader::YamlFrameReader;
 
+#[cfg(target_os = "windows")]
+const PLATFORM: &str = "win";
+#[cfg(target_os = "linux")]
```

Comment:
Do we alienate non-linux UNIX systems into `other` category?
Perhaps, we can use `cfg(windows)`, `cfg(unix)`, and `cfg(target_os="macos")` instead, see [target flags](https://doc.rust-lang.org/reference/attributes.html#conditional-compilation) for more info

Reply:
Yea - you're right. I'll update it to be unix.